### DISCUSSION TITLE
How can I use dbus with private-tmp and netns?

### DISCUSSION THREAD

**Discussion by amano-kenji** [netblue30/firejail#5279] [Q&A] [answered]
With private-tmp and `netns some-ns`, dbus abstract socket is gone, and the dbus socket in /tmp is also gone.

Thus, `dbus-filter user` can't do anything.

**Comment by rusty-snake** [comment, marked as answer]
> How can I use dbus with private-tmp and netns?

Search for similiar discussions: #4737

**Reply by amano-kenji** [reply]
Right.
